packages/zowe-explorer/__tests__/__unit__/utils/SettingsConfig.unit.test.ts

```diff
@@ -13,6 +13,11 @@ import { SettingsConfig } from "../../../src/utils/SettingsConfig";
 import * as vscode from "vscode";
 import { Gui } from "@zowe/zowe-explorer-api";
 
+afterEach(() => {
+    jest.clearAllMocks();
+    jest.resetAllMocks();
```

Comment:
Curious if `jest.restoreAllMocks()` will work here (replacing both,`clear` and `reset`) since we are (or should be) spying on every test (either manually or in the `beforeEach`)?

No change required though :yum: